### ISSUE TITLE
TileGrid position has some unknown transformation (and also we can't move)

### ISSUE THREAD

**Issue by ladyada** [adafruit/circuitpython#1613]
this could be me not understanding tilegrid but when i subclass it to create shapes with outlines, it displays in the right spot, but the position is weirdly transformed?

download this:
https://github.com/ladyada/Adafruit_CircuitPython_Display_Shapes/blob/master/adafruit_display_shapes/rect.py
then run:

```import displayio
import board
from adafruit_display_shapes.rect import Rect

# Make the display context
splash = displayio.Group()
board.DISPLAY.show(splash)
rect = Rect(10, 10, 20, 20, fill=0xFF0000, outline=0x0000FF)
splash.append(rect)

print(rect.position)

while True:
    pass
```

red rect with blue outline appears, but position prints out as `(13504, 8192)` not `(10,10)`

Running `Adafruit CircuitPython 1ba7610 on 2019-03-01;`

**Comment by zencuke** [comment]
I don't know if this adds any information but as long as I did the test I thought I would add it here. I was looking at this code as a tilegrid example (I'm hoping to create similar app specific drawing primitives) and got curious. I added a bunch of debug print statements to rect.py:rect.__INIT__ to see where it happened. The position value was initially (0,0) and changed after the line 

> self._bitmap =displayio.Bitmap(...) 

to the same value printed in the above wrapper example code.
